### PR TITLE
Change the donutstate with empty service

### PR DESCRIPTION
**Pull request by pablothedude**
If a service has no entities the donut should reflect that. So it
must not look if any progess so far has been made.

### CODE REVIEW

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/story/show/164241210

File: src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php [hunk 1]
```diff
@@ -194,6 +194,8 @@ private function getStates(Service $service, ServiceStatusService $serviceStatus
         return $states;
     }
 
+
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php]: Nitpick: Remove exessive whitespace

Comment (pablothedude) [reply src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php]: Done

**Comment by pablothedude** [comment]
Because there isn't much logic in the api controller I think I could do without a webtest. I think the tests in the `ServiceStatusAssemblerTest` will be sufficient for now.

**Pull request merged**